types/module/module.go

```diff
@@ -317,6 +320,11 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 		}
 	}
 
+	// a chain must initialize with a non-empty validator set
+	if len(validatorUpdates) == 0 {
+		panic(fmt.Sprintf("validator set is empty after InitGenesis, please ensure at least one validator is initialized with a delegation greater than or equal to the DefaultPowerReduction (%d)", sdk.DefaultPowerReduction))
```

Comment:
```suggestion
		panic("validator set is empty after InitGenesis, please ensure at least one validator is initialized with a delegation greater than or equal to the PowerReduction parameter set in staking")
```
This has become an on-chain parameter

Reply:
I think we revert it back due to #9447.

Reply:
Ah! I got confused by the [leftover changes](https://github.com/cosmos/cosmos-sdk/pull/10248)